io.catenax.failure_pattern/1.0.0/FailurePattern.ttl

```diff
@@ -0,0 +1,278 @@
+#######################################################################
+# Copyright (c) 2022, 2024 Robert Bosch GmbH
```

Comment:
since this is first version, only 2024 is required.

Reply:
Oh ja surem, fixed that as well